src/controller/java/src/chip/devicecontroller/model/ChipEventPath.java

```diff
@@ -0,0 +1,77 @@
+/*
+ *   Copyright (c) 2022 Project CHIP Authors
+ *   All rights reserved.
+ *
+ *   Licensed under the Apache License, Version 2.0 (the "License");
+ *   you may not use this file except in compliance with the License.
+ *   You may obtain a copy of the License at
+ *
+ *       http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *   Unless required by applicable law or agreed to in writing, software
+ *   distributed under the License is distributed on an "AS IS" BASIS,
+ *   WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *   See the License for the specific language governing permissions and
+ *   limitations under the License.
+ *
+ */
+package chip.devicecontroller.model;
+
+import java.util.Locale;
+import java.util.Objects;
+
+/** An event path that should be used for requests. */
+public class ChipEventPath {
+  private ChipPathId endpointId, clusterId, eventId;
+
+  private ChipEventPath(ChipPathId endpointId, ChipPathId clusterId, ChipPathId eventId) {
+    this.endpointId = endpointId;
+    this.clusterId = clusterId;
+    this.eventId = eventId;
```

Comment:
Please have urgent boolean here. in spec, we have record the urgent flag inside path.
https://github.com/CHIP-Specifications/connectedhomeip-spec/blob/master/src/data_model/Encoding-Specification.adoc#encoding-EventPathIB